server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DBStructureComparisonFlowableTask.java

```diff
@@ -15,50 +15,127 @@
  */
 package com.oceanbase.odc.service.flow.task;
 
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.Validate;
 import org.flowable.engine.delegate.DelegateExecution;
+import org.springframework.beans.factory.annotation.Autowired;
 
-import com.oceanbase.odc.service.flow.task.model.DBStructureComparisonTaskResult;
+import com.oceanbase.odc.core.shared.constant.FlowStatus;
+import com.oceanbase.odc.core.shared.constant.TaskStatus;
+import com.oceanbase.odc.metadb.task.TaskEntity;
+import com.oceanbase.odc.service.flow.task.model.DBStructureComparisonParameter;
+import com.oceanbase.odc.service.flow.util.FlowTaskUtil;
+import com.oceanbase.odc.service.structurecompare.StructureComparisonContext;
+import com.oceanbase.odc.service.structurecompare.StructureComparisonService;
 import com.oceanbase.odc.service.task.TaskService;
 
+import lombok.extern.slf4j.Slf4j;
+
 /**
  * @author jingtian
  * @date 2024/1/9
- * @since
+ * @since ODC_release_4.2.4
  */
-public class DBStructureComparisonFlowableTask extends BaseODCFlowTaskDelegate<DBStructureComparisonTaskResult> {
+@Slf4j
+public class DBStructureComparisonFlowableTask extends BaseODCFlowTaskDelegate<Void> {
+    private volatile StructureComparisonContext context;
+    @Autowired
+    private StructureComparisonService service;
+
     @Override
-    protected DBStructureComparisonTaskResult start(Long taskId, TaskService taskService, DelegateExecution execution)
+    protected Void start(Long taskId, TaskService taskService, DelegateExecution execution)
             throws Exception {
+        TaskEntity taskEntity = taskService.detail(taskId);
+        if (taskEntity == null) {
+            throw new IllegalStateException("Can not find task entity by id " + taskId);
+        }
+        log.info("Structure comparison task starts, taskId={}, activityId={}", taskId,
```

Comment:
so, what are this log messages print to?

Reply:
to default appender? and the user can not access it directly in task detail panel?

Reply:
> to default appender? and the user can not access it directly in task detail panel?
yes, Because I see that this log is printed in all FlowableTask#start